app/scrapers/youtube.ts

```
@@ -0,0 +1,232 @@
+            .progress_url);
+          metadata.downloadUrl = downloadUrl;
+          return createSuccessResponse(metadata as YouTubeDownloadResult);
+        } catch (timeoutError) {
+          return createErrorResponse(timeoutError.message, {
+            type: ScraperErrorType.NETWORK_ERROR
+          });
+        }
+        
+        metadata.downloadUrl = downloadUrl;
+        
+        return createSuccessResponse(metadata as YouTubeDownloadResult);
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement, ...

... this statement, ...

... this statement, ...

... this statement, ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)